hw/syn/tools/dc/parse-syn-report.py

```diff
@@ -89,7 +89,10 @@ def _rel_err(val, ref):
     """
     Calculate relative error with respect to reference
     """
-    return abs(val - ref) / ref
+    if ref == 0.0 or np.isnan(ref) or np.isnan(val):
+        return np.nan
```

Comment:
I just noticed that you're using numpy here, which is not in the `python-requirements.txt` file currently. Please either add it there, or remove the dependency. (It's actually a quite heavy dependency, it's probably worth thinking about using normal Python functionality instead.